client/src/components/About/about.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+
+class About extends Component {
+  render() {
+    const imageStyle = {
```

Comment:
move this out of the render function. It will make it cleaner imo. Then just reference it as this.imageStyle cause this is techniquely the component.